models/intel/index.md

```diff
@@ -4,9 +4,8 @@ OpenVINO&trade; toolkit provides a set of pre-trained models
 that you can use for learning and demo purposes or for developing deep learning
 software. Most recent version is available in the [repo on Github](https://github.com/openvinotoolkit/open_model_zoo).
 
-The models can be downloaded via Model Downloader
-(`<OPENVINO_INSTALL_DIR>/deployment_tools/open_model_zoo/tools/downloader`).
-They can also be downloaded manually from [01.org](https://download.01.org/opencv).
+The models can be downloaded via Model Downloader and other automation tools.
```

Comment:
That's not really true. You don't use _other_ tools to download models.

Reply:
changed